config/eirini/_ytt_lib/eirini/rendered.yml

```diff
@@ -1008,8 +1008,10 @@ kind: Deployment
 metadata:
   annotations:
     kbld.k14s.io/images: |
-      - Metas: null
-        URL: index.docker.io/eirini/opi@sha256:c7c60d73c89db5c1a74019b86db0fb93686ed5572984411f1d4a027681aec457
+      - Metas:
+        - Type: preresolved
+          URL: index.docker.io/relintdockerhubpushbot/cf-for-k8s-eirini-opi@sha256:1f08f8ca4e6981034043612159f7268e43e84a9d5d000c6c88f802c54fc5d265
+        URL: index.docker.io/relintdockerhubpushbot/cf-for-k8s-eirini-opi@sha256:1f08f8ca4e6981034043612159f7268e43e84a9d5d000c6c88f802c54fc5d265
```

Comment:
Looks like duplicates here?

Reply:
Hmmm...as I understand it these annotations are generated by kbld so I wonder whats up with that?  Will investigate

Reply:
https://kubernetes.slack.com/archives/CH8KCCKA5/p1597125371477200

Reply:
According to Dmitriy this is not a duplicate it is intended behavior.  See slack thread.